lib/bolt/outputter/human.rb

```diff
@@ -136,6 +138,14 @@ def print_task_info(task)
         task_info << "\n\n"
         task_info << "USAGE:\n#{usage}\n\n"
         task_info << "PARAMETERS:\n#{pretty_params}\n" unless pretty_params.empty?
+        task_info << "MODULE:\n"
+
+        path = task['files'][0]['path'].chomp("/tasks/#{task['files'][0]['name']}")
+        task_info << if path.include?(Bolt::PAL::MODULES_PATH)
```

Comment:
Something like `File.realdirpath(path) == File.realdirpath(Bolt::PAL::MODULES_PATH)` might be more robust.

Reply:
I'd prefer to avoid another assumption that the task is in the top level of the tasks directory. `.start_with?` ?

Reply:
Sure, can drop the chomp and use `start_with?`.

Reply:
Presumably Ruby will be consistent about constructing absolute paths, so we probably don't need extra normalization.